### DISCUSSION TITLE
Unable to connect to staking Coins

### DISCUSSION THREAD

**Discussion by StephanDietl** [iotaledger/firefly#2222] [Help & Support] [answered]
I staked all of the IOTA tokens which are listed in firefly since the very first day.
It worked perfectly and the Assembly and Shimmer earnings were tracked well.

Since about 6 days firefly is not able to establish a connection to the staking account.
Networking configuration is and was set to manually - node: https://chrysalis-nodes.iota.cafe is primary and was set to "participation"

I am not able to recheck that - cause while clicking on info there is no connection.

Do you have a glue whats wrong - or facing the same issue - is there a new node which i should add?
Unfortunately setting to automatic mode is also not working.

Thanks for your kind help.

**Comment by JannemanDev** [comment]
Set node selection to Manual and use `https://mainnet-node.tanglebay.com` as Primary node.

**Comment by t00k** [comment, marked as answer]
Also, if the above doesnt work. Try creating a new profile using the same 24 words/stronghold/ledger as your first one.
Be sure to update to version 1.3.3 since it has fixed some bugs, that you might be affected of.

Please remember to NEVER share your seed with ANYONE. A lot of scammers lurking around in crypto.
